compiler/rustc_resolve/src/late.rs

```diff
@@ -1224,7 +1224,7 @@ impl<'a: 'ast, 'b, 'ast, 'tcx> LateResolutionVisitor<'a, 'b, 'ast, 'tcx> {
             lifetime_ribs: Vec::new(),
             lifetime_elision_candidates: None,
             current_trait_ref: None,
-            diagnostic_metadata: Box::new(DiagnosticMetadata::default()),
+            diagnostic_metadata: Box::<DiagnosticMetadata<'_>>::default(),
```

Comment:
i dislike this lol

Reply:
I can remove the turbofish if you want^^

Reply:
Or maybe just `Default::default()` lol